docs/getting_started/basics/debug.md

```diff
@@ -28,8 +31,10 @@ Shift provides many ways of executing workflows in a modular way. For instance,
 
 Additionally, Shift allows to add breakpoints to the nodes. When a node is flagged with a breakpoint, the execution of the graph will stop once it reaches that node. To add a breakpoint, `Right Click` on the node and select the *Breakpoint* option. To remove it, follow the same steps. 
 
-![Breakpoint](../../images/breakpoint.gif)
-
+<figure>
+      <img src="images/breakpoint.gif" alt="Breakpoint">

```

Comment:
In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

Reply:
I think this one has not been updated.

Reply:
Now is updated